src/views/FileListInfo.vue

```diff
@@ -20,7 +20,8 @@
 
 <script>
 import { generateUrl } from '@nextcloud/router'
-import { NcButton } from '@nextcloud/vue'
+// As it's the only nextcloud-vue component used in collectives-files.js, import it separately to keep asset small
+import NcButton from '@nextcloud/vue/dist/Components/NcButton.js'
```

Comment:
I thought this was no longer necessary due to tree-shaking? I changed all imports to `'@nextcloud/vue'` a while back :angry: